spec/components/record/mods_bibliographic_component_spec.rb

```diff
@@ -0,0 +1,40 @@
+# encoding: UTF-8
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Record::ModsBibliographicComponent, type: :component do
+  include ModsFixtures
+
+  describe "Object bibliographic" do
+    let(:document) { SolrDocument.new(modsxml: mods_everything) }
+    let(:component) { described_class.new(document:) }
+
+    before do
+      render_inline(component)
+    end
+
+    it "should display titles" do
```

Comment:
Using "Should" is typically a violation of `RSpec/ExampleWording` I know it's not enabled on this project, but I think it's best to follow that as we typically use it everywhere.